src/sql/base/browser/ui/panel/panel.component.ts

```diff
@@ -85,7 +85,13 @@ let idPool = 0;
 	`
 })
 export class PanelComponent extends Disposable implements IThemable {
-	@Input() public options?: IPanelOptions;
```

Comment:
have you checked all other places that are using panel component? they might be still setting the options as input.